CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ All notable changes to the "azure-appservice" extension will be documented in th
 - Create deployment slots from configuration sources
 - Show current running state of web apps
 - Improve resource deployment scripts
-- Leverage npm package "vscode-azureextensionui"
+- UI improvements including "Creating..." placeholder nodes, "Load More..." when there are a lot of web apps, ect.
```

Comment:
ect?

Reply:
Dang it, I fixed this almost immediately 😞

Reply:
Suggest:  UI improvements, including "Creating..." status, "Load More" button and others